discovery/marathon/marathon.go

```diff
@@ -49,7 +49,8 @@ const (
 	portIndexLabel model.LabelName = metaLabelPrefix + "port_index"
 	// taskLabel contains the mesos task name of the app instance.
 	taskLabel model.LabelName = metaLabelPrefix + "task"
-
+	// hostLabel contains the mesos task host address of the app instance
```

Comment:
full stop

Reply:
I believe the full stop was used to group port labels together. 

So the stop probably should be after the hostLabel Line correct?

See the following:
https://github.com/prometheus/prometheus/blob/master/discovery/marathon/marathon.go#L52

Reply:
The style in Go is to have comments as full sentences, which includes a full stop. So all of these should have a full stop.